### PR TITLE
[WIP] Add ocp-build 1.99.21 for all ocaml versions

### PR DESCRIPTION
**Pull request by lefessan**

### CODE REVIEW

File: packages/ocp-build/ocp-build.1.99.21/opam [hunk 1]
```diff
@@ -16,7 +16,7 @@ depends: [
   "ocamlfind"
   "cmdliner" {>= "1"}
   "re" {>= "1.7.3"}
-  "seq" { ocaml:version < "4.07.0"}
+  "seq" { ocaml < "4.07.0"}
```
Comment (mseri) [inline packages/ocp-build/ocp-build.1.99.21/opam]: What about 
```suggestion
  ("seq" | "ocaml" {>= "4.08"})
```

Comment (lefessan) [reply packages/ocp-build/ocp-build.1.99.21/opam]: Thanks, but already fixed... normally :-) (thx to @AltGr for the trick)

**Comment by mseri** [comment]
The failures are:
- opam-core.2.0~alpha5 to beta3.1: need an upper bound on ocaml < 4.07, not your fault
- typerex: missing zlib.h and bzlib.h, not your fault
- wxocaml: missing packages for depexts, not your fault

For me it is good to go. I appreciate if you can add those upper bounds on opam core, otherwise I will do it in the next days

**Comment by lefessan** [comment]
Thanks, I tried to fix most revdeps, let's see what the CI thinks about it :-)

**Comment by lefessan** [comment]
Everything looks ok now !

**Comment by mseri** [comment]
Awesome, thanks!

**Pull request merged**